recipes/r-emmeans/meta.yaml

```diff
@@ -1,71 +0,0 @@
-{% set version = '1.1.3' %}
```

Comment:
Why is this file removed? Please rebase onto master if possible

Reply:
rebasing wasn't easily possible as the r-emmeans was replaced by libmed. So I simple reset and added libmed again. But I wonder why are these r-libraries are needed???

Reply:
My main concern is that if this gets merged an the r library hasn't been moved to a feedstock then that commit would have removed it and the feedstock wouldn't have been made.

Reply:
now I understand. thanks for clarification.